release_notes_generator/action_inputs.py

```diff
@@ -98,11 +101,24 @@ def is_from_tag_name_defined() -> bool:
         return value.strip() != ""
 
     @staticmethod
-    def get_chapters_json() -> str:
+    def get_chapters() -> Optional[list[dict[str, str]]]:
         """
-        Get the chapters JSON from the action inputs.
+        Get list of the chapters from the action inputs. Each chapter is a dict.
         """
-        return get_action_input(CHAPTERS)
+        # Get the 'chapters' input from environment variables
+        chapters_input: str = get_action_input(CHAPTERS, default="")
+
+        # Parse the YAML input
```

Comment:
Please extend this comment, so it more reflects the logic inside this method.

Reply:
Fixed.